docs/configuration.asciidoc

```diff
@@ -360,4 +394,21 @@ You can use this setting to dynamically disable Elastic APM at runtime.
 |============
 | Default                          | Type                | Dynamic
 | `true` | Boolean | true
-|============
\ No newline at end of file
+|============
+
+[float]
+[[config-session-sample-rate]]
+==== `session_sample_rate` (added[0.9.0])
+
+By default, the agent will sample all signals generated by your application (e.g. spans, metrics, & logs).
+To reduce overhead and storage requirements, you can set the sample rate to a value between 0.0 and 1.0. When reduced below 1.0, data will be sampled per session.
+This is so context in a given session isn't lost.
+You can use this setting to dynamically disable Elastic APM at runtime.
```

Comment:
Should we explain that to do this, you set the sample rate to zero? for example:

```suggestion
You can use this setting to dynamically disable Elastic APM at runtime by setting the sample rate to `0`.
```